build.gradle

```diff
@@ -5,7 +5,7 @@ apply plugin: 'spring-boot'
 apply plugin: 'groovy'
 apply plugin: 'signing'
 
-version = '2.0.1'
+version = '0.7.1.2'
 group = 'com.bericotech'
```

Comment:
I'm actually not too familiar with nexus requirement, must it have an actual website behind it?